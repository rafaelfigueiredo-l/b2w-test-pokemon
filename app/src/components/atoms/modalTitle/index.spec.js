import ModalTitle from "./";

describe("<ModalTitle />", () => {
  const component = <ModalTitle>Texto</ModalTitle>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly ModalTitle component", () => {
    expect(wrapper.find(".modalTitle")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly ModalTitle component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".modalTitle").children()).toHaveLength(1);
  });
});
