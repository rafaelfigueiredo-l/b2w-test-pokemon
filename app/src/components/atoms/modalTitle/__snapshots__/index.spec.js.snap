// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalTitle /> should mount correctly ModalTitle component 1`] = `
<styled.h1
  className="modalTitle"
>
  <h1
    className="sc-AxjAm jWlVxK modalTitle"
  >
    Texto
  </h1>
</styled.h1>
`;
